src/native.rs

```diff
@@ -870,6 +870,45 @@ impl HasContext for Context {
         );
     }
 
+    unsafe fn get_active_attributes(&self, program: Self::Program) -> u32 {
+        let gl = &self.raw;
+        let mut count = 0;
+        gl.GetProgramiv(program, ACTIVE_ATTRIBUTES, &mut count);
+        count as u32
+    }
+
+    unsafe fn get_active_attrib(
```

Comment:
nit: could we name it `get_active_attribute`?

Reply:
Absolutely! I should have noticed that that was the convention.